tools/wptrunner/wptrunner/wpttest.py

```diff
@@ -554,6 +554,10 @@ def update_metadata(self, metadata):
             reference.update_metadata(metadata)
         return metadata
 
+    def get_viewport_size(self, override):
```

Comment:
What is `override` for, if both implementations assert it is `None`?

Reply:
It's basically just scafolding for the possibility of implementing this in the future. My guess is that different page sizes will be more important for paginated media than for visual.

Reply:
Hrm, ok. I prefer not supporting code that isn't used, but I'll leave it up to you to decide.